Package.swift

```diff
@@ -270,7 +270,7 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
     .package(url: "https://github.com/apple/swift-tools-support-core.git", branch: relatedDependenciesBranch),
     .package(url: "https://github.com/apple/swift-argument-parser.git", from: "1.2.2"),
     .package(url: "https://github.com/apple/swift-syntax.git", branch: relatedDependenciesBranch),
-    .package(url: "https://github.com/apple/swift-crypto.git", from: "2.5.0"),
+    .package(url: "https://github.com/apple/swift-crypto.git", from: "3.0.0"),
```

Comment:
[Question] what's the logic to decide the range of versions?
should we align it with the setting in [swift-package-manager](https://github.com/apple/swift-package-manager/blob/main/Package.swift#L739)
`.upToNextMinor(from: "3.0.0")`

Reply:
This one being broad is fine, means we'll only need to bump it in SwiftPM in the future and keep it here as is until 4.0